org.opent2t.sample.hub.superpopular/com.wink.hub/js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "opent2t-wink-hub",
+  "version": "1.0.0",
+  "description": "Hub schema and helper modules for Open Translators to Things",
+  "author": "OpenT2T",
```

Comment:
None of the other translators have this... I suggest deleting. Also, other translators have he "homepage" element. I suggest referencing https://github.com/openT2T/translators/blob/dev/org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/package.json and following as closely as possible.